### DISCUSSION TITLE
Why is lara-dark-teal so washed out in PrimeNG 17 compared to 16?

### DISCUSSION THREAD

**Discussion by horizoncarlo** [primefaces/community#666] [PrimeNG] [answered]
What was the reasoning for making the `lara-dark-teal` theme so gray and washed out? I'd prefer to stick on an actual theme to get updates instead of having to revert a local version.

## PrimeNG 16:
![ang16-main](https://github.com/primefaces/community/assets/65215473/643ba387-0ce6-4301-841f-9445c8e07a44)

## PrimeNG 17:
![ang17-main](https://github.com/primefaces/community/assets/65215473/b3eb565c-f1e9-496d-b36c-9039f01618f4)

**Comment by cetincakiroglu** [comment, marked as answer]
Hi,

We've updated the lara themes in v17. It's made by design thanks a lot for the feedback.

**Comment by horizoncarlo** [comment]
Okay, well put me down in the camp of "not liking the change", although it's too late to vote :)
